[class] Replace 'could' and 'might' (#4361)
cplusplus/draft: source/classes.tex

In a \grammarterm{member-declarator} for a bit-field,
the \grammarterm{constant-expression} is parsed as
the longest sequence of tokens

> that would syntactically form a \grammarterm{constant-expression}.

Reviewer: I think we can avoid the hypothetical here (and in other cases where we describe tentative parses):
```suggestion
that matches the syntax of a \grammarterm{constant-expression}.
```

Author: fixed

\begin{note}
A
\tcode{return}
statement\iref{stmt.return} in a destructor does not directly return to the
caller; before transferring control to the caller, the destructors for the

> members and bases (if any) are called.

Reviewer: The pre-existing wording is unnecessarily verbose. I think the "does not" might be confusing in the case where there are no members or bases. How about replacing this with:

"A `return` statement ([stmt.return]) in a destructor calls the destructors for the members and bases (if any) before transferring control to the caller."

Author: fixed

in a
\grammarterm{conversion-function-id}
is the longest sequence of

> tokens that would possibly form a \grammarterm{conversion-type-id}.

Reviewer: ```suggestion
tokens that match the syntax of a \grammarterm{conversion-type-id}.
```

Author: "that matches" (singular); it's referring to the sequence

fixed

requiring a constant expression\iref{expr.const}
and in constant initialization\iref{basic.start.static}.
\begin{note}

> It is possible for copy elision to be performed

Reviewer: ```suggestion
It is possible that copy elision is performed
```
